magenta/music/musicxml_parser.py

```diff
@@ -486,6 +514,55 @@ def _parse_forward(self, xml_forward):
                * self.state.seconds_per_quarter)
     self.state.time_position += seconds
 
+  def _check_time_signature(self):
```

Comment:
could you rename this to _fix_time_signature? I think that makes it a little more clear that this method actually modifies things.